### PR TITLE
Feature/238 gender neutral website update

### LINKED ISSUE #238: Gender-neutral website update

### PR DESCRIPTION
**Pull request by tgraupne**
Gender-neutral update of the website itself and linked assets.

After thoroughly reviewing all of our sites, we worked on a gender-neutral branding. This pr presents all the corresponding updates.

Closes #238 #99

### CODE REVIEW

File: src/components/DesktopMenu.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function DesktopMenu() {
 
   const Menu = () => (
     <Box className="flex w-1/2" flexDirection="column">
-      <ul className="font-exo2 flex justify-around text-lg font-semibold mt-5 mr-5 -ml-10">
+      <ul className="font-exo2 flex justify-around text-base font-semibold mt-4 mr-4 -ml-16">
```
Comment (Maurice22) [inline src/components/DesktopMenu.jsx:17]: In my opinion, the buttons are too small when using this font size

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -49,8 +49,8 @@ export default function Sidebar(props) {
       <MenuItem to="/ask-for-help">{t('components.sidebar.askForHelp')}</MenuItem>
       <MenuItem to="/overview">{t('components.sidebar.overview')}</MenuItem>
       {menuProps.isLoggedIn && <MenuItem to="/dashboard">{t('components.sidebar.yourOverview')}</MenuItem>}
-      <MenuItem to="/faq">{t('components.sidebar.FAQs')}</MenuItem>
       <MenuItem to="/security-tips">{t('components.sidebar.safety')}</MenuItem>
```
Comment (Maurice22) [inline src/components/Sidebar.jsx:52]: This text is not changed to "Sicherheitshinweise". Should be changed for consistency

File: src/views/FAQ.jsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ export default function FAQ() {
   const QA = ({ question, children }) => (
     <>
       <h2 className="text-xl font-teaser mt-8">{question}</h2>
-      <p className="font-open-sans">{children}</p>
+      <p className="font-open-sans text-justify">{children}</p>
```
Comment (Maurice22) [inline src/views/FAQ.jsx:29]: Nice!

File: src/views/Security.jsx [hunk 4]
```diff
@@ -79,55 +81,71 @@ export default () => (
...
 
-    <div className="mt-4">
+    <div className="p-4 bg-kaki font-open-sans mt-5">
+      <div className="font-bold">Hinweis</div>
+      Trotz sorgfältiger inhaltlicher Kontrolle übernehmen wir keine Haftung für die Aktualität und Korrektheit der
+      Informationen auf dieser Seite und der Inhalte externer Links. Für den Inhalt der verlinkten Seiten sind
+      ausschließlich deren Betreiber*innen verantwortlich.
+    </div>
+
+    <div className="mt-5">
```
Comment (Maurice22) [inline src/views/Security.jsx:148]: Why did we move this upwards? For me it feels out of place as the margin of the text is different to all surrounding text

File: src/views/Main.jsx [hunk 5]
```diff
@@ -59,40 +59,29 @@ export default function Main() {
...
+          <div className="m-1 md:m-4" />
+          <Link
+            data-cy="cta-want-to-help"
+            to="/overview"
+            className="flex justify-center items-center rounded text-white py-3 pl-1 pr-3 btn-main bg-secondary md:flex-1 hover:opacity-75"
+            onClick={() => fb.analytics.logEvent('button_want_to_help')}
+          >
+            <img className="h-8 mr-1" src={require('../assets/hero_white.svg')} alt="" />
+            {t('views.main.buttons.wantToHelp')}
+          </Link>
```
Comment (Maurice22) [inline src/views/Main.jsx:80]: Why did we remove the bold words here? For me, with this much text, it helps when I have some anchors.

File: src/views/Security.jsx (hunk 4 above)
Comment (tgraupne) [reply src/views/Security.jsx:148]: The reason was a misunderstanding. I reverted this change.

File: src/components/entry/Entry.css [hunk 6]
```diff
@@ -1,3 +1,3 @@
 .entry {
-    hyphens: auto;
+    
 }
```
Comment (Maurice22) [inline src/components/entry/Entry.css:3]:
```suggestion
    
}

```

File: public/locales/de/translation.json [hunk 7]
```diff
@@ -11,40 +11,42 @@
...
       "importantNotes": {
         "title": "WICHTIGER HINWEIS",
+        "cautious": {
+          "title": "Umsichtig",
+          "text": "Helft nicht, wenn Ihr einer Risikogruppe angehört! Helft nicht, wenn Ihr Sorge habt, selbst eine Ansteckungsgefahr darzustellen!"
+        },
         "local": {
           "title": "Lokal",
-          "text": "Helft in eurer Nachbarschaft, zum Beispiel euren Haus-Mitbewohner*innen."
+          "text": "Helft in eurer Nachbarschaft, zum Beispiel euren Hausmitbewohner*innen."
```
Comment (Maurice22) [inline public/locales/de/translation.json:20]:
```suggestion
          "text": "Helft in Eurer Nachbarschaft, zum Beispiel Euren Hausmitbewohner*innen."
```

File: public/locales/de/translation.json [hunk 8]
```diff
@@ -11,40 +11,42 @@
...
         },
         "consistent": {
           "title": "Konsistent",
           "text": "Helft wenigen, aber das konsistent. Sucht euch zum Beispiel eine Familie und helft nur dieser."
```
Comment (Maurice22) [inline public/locales/de/translation.json:24]:
```suggestion
          "text": "Helft wenigen, aber das konsistent. Sucht Euch zum Beispiel eine Familie und helft nur dieser."
```

**Pull request merged**